tools/executiontester/executiontester.go

```diff
@@ -80,27 +83,45 @@ func main() {
 	if _, err := fpToken.CreateToken(assetID, debtorAddr, creditorAddr, total, risk, open, exp); err != nil {
 		log.L().Fatal("Failed to create fp token", zap.Error(err))
 	}
-
+	// create fpToken with assetID
 	contractAddr, err := fpToken.TokenAddress(assetID)
 	if err != nil {
 		log.L().Fatal("Failed to get token contract address", zap.Error(err))
 	}
-
+	// create erc721 token with tokenID using assetID
+	if _, err := erc721Token.CreateToken(assetID, creditorAddr); err != nil {
```

Comment:
Don't you need a new assetID for erc721Token?

Reply:
added a new id for erc721